main.py

```diff
@@ -9,22 +9,42 @@
 import openai
 from openai.error import Timeout
 
-
-def main(api_key: str, model: str, query_words: list, chat_timeout: int):
+DEFAULT_OPENAI_BASE_API = "https://api.openai.com/v1"
+DEFAULT_MODEL = "gpt-3.5-turbo"
```

Comment:
I'd say just keep these two as defaults in the arguments - they aren't used anywhere else ☺️

Reply:
Good point, I have extracted them because I had first some if-else logic but then realize that the OPENAI_API_KEY is a requirement by the OpenAI package to be set. Hence I deleted my if-else logic